bin/templates/project/build.gradle

```diff
@@ -19,13 +19,13 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.4.10'
```

Comment:
Personally I don't mind updating the complete toolset in one PR, but one could argue that either it's listed in the PR description and title or updated in a separate PR.

Reply:
Your idea is correct, but this PR update any related to Gradle, I update the title to all updates affected and also I put the details in description filed.

Reply:
Google does not follow traditional semver rules and minor releases can include breaking changes, therefore we cannot upgrade the minor version of Kotlin without also introducing a breaking change.

Reply:
I revert the kotlin version to 1.3.50.

So the other should keep updated to dismiss the sync at android studio